tests/test_html.py

```diff
@@ -480,3 +485,19 @@ def test_del_qparam_default(request):
     ret = mod.del_qparam(None, 'a')
     assert 'a=1' not in str(ret)
     assert 'b=2' in str(ret)
+
+
+@pytest.mark.parametrize('parts expected with_scheme'.split, (
+    ['https://outernet.is:80/', '//outernet.is/?query', False],
+    ['https://outernet.is:443/', '//outernet.is/?query', False],
+    ['https://outernet.is:1234/', '//outernet.is:1234/?query', False],
+    ['https://outernet.is:80/', 'https://outernet.is/?query', True],
+    ['http://outernet.is:80/', 'http://outernet.is/?query', True],
+))
+@mock.Mock(MOD + 'request.urlparts')
+@mock.patch(MOD + 'quoted_url', return_value='/?query')
```

Comment:
It was an arbitrary value i was using, it could return any value that can be reproduced. It requires a return value because it's used in the `full_quoted_url` function.